src/address_book.test.js

```diff
@@ -35,17 +35,21 @@ describe('Address book', () => {
   const filePath = path.relative(process.cwd(), path.join(__dirname, '/../utils/files/address_book_test.csv'))
   const ENSName = 'francotest.eth'
 
-  test('Addrress book', async (done) => {
-    console.log('Addrress book')
+  test('Address book', async (done) => {
+    console.log('Address book')
```

Comment:
This can be removed now.

Reply:
I use that console.log to set the title for each description